Gemfile

```diff
@@ -14,3 +14,5 @@ gemspec
 # To use a debugger
 gem "byebug", group: [:development, :test]
 gem "appraisal", github: "excid3/appraisal", branch: "fix-bundle-env"
+# Temporary patch for the big mimemagic SNAFU
+gem "mimemagic", github: "mimemagicrb/mimemagic", ref: "01f92d86d15d85cfd0f20dabd025dcbd36a8a60f"
```

Comment:
Now that the new Rails releases are out, I think you can drop this and update the Rails versions: https://weblog.rubyonrails.org/2021/3/26/marcel-upgrade-releases/

Reply:
Ah yes indeed -- we are green again now.
Important to have the rails_master lock to be happy was to remove mini_portile2 :)